<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HostInfoPopup</class>
 <widget class="QWidget" name="HostInfoPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>425</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="labelAddr">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelAddrField">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonStatistic">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>141</width>
     <height>23</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Statistic</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHostname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBytesSnt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BytesSnt: 5 MiB</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBytesRcv">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>BytesRcv: 14 KiB</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPacketsSnt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>251</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PcktsSnt: 0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelPacketsRcv">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>301</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>PcktsRcv: 0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
